<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QDialog" name="RegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>281</width>
     <height>76</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelUsername">
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelPassword">
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditPassword"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelscndPassword">
      <property name="text">
       <string>Re-type password</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEditscndPassword"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditUsername"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelMonth">
      <property name="text">
       <string>Month</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="labelDay">
      <property name="text">
       <string>Day</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditDay"/>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineEditYear"/>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="labelYear">
      <property name="text">
       <string>Year</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="comboBoxmonth">
      <item>
       <property name="text">
        <string>January</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>February</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>March</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>April</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>May</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>June</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>July ðŸ¤®</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>August</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>September</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>October ðŸŽƒðŸ’•</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>November</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>December</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>181</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Favorite Genres</string>
   </property>
   <widget class="QCheckBox" name="checkBoxAction">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Action</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxCom">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Comedy</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxRom">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Romance</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxHorror">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Horror</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxDrama">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>38</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Drama</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxOther">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Other</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonRegister">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>290</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QLabel" name="ageError">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>120</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>* You must be over 18 to register.</string>
   </property>
  </widget>
  <widget class="QLabel" name="radioButtonError">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>* Pick a gender and/or an account type.</string>
   </property>
  </widget>
  <widget class="QLabel" name="checkboxError">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>* Pick a favorite genre(s).</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Gender</string>
        </property>
        <widget class="QRadioButton" name="radioButtonMale">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>69</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text">
          <string>Male</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButtonFemale">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>69</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text">
          <string>Female</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Account type</string>
        </property>
        <widget class="QRadioButton" name="radioButtonUser">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>69</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text">
          <string>User</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButtonAdmin">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>69</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Admin</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>99</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="usernameError">
      <property name="text">
       <string>* Username unavailable.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="passwordError">
      <property name="text">
       <string>* Password unavailable.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="scndPasswordError">
      <property name="text">
       <string>* Unmatching Password.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
